utils/link_preview_util.ts

```diff
@@ -0,0 +1,8 @@
+const linkExpanderUserAgents = [
+    'Discordbot',
+    'Slackbot-LinkExpanding'
```

Comment:
are there other strings we need to include here? i remember looking into this in the past to support fb messenger link previews, though i'm guessing it's a different protocol? https://ogp.me/

Reply:
i found this thingy which might be helpful https://github.com/monperrus/crawler-user-agents/blob/master/crawler-user-agents.json

Reply:
So interestingly, it seems like Messenger only supports title + thumbnail in its link previews:
![image](https://github.com/downforacross/downforacross.com/assets/14168765/97b7e547-3027-4063-850c-3f30dba065ee)

So as a hack I ended up just cramming everything into the title (line breaks don't seems to work)
![image](https://github.com/downforacross/downforacross.com/assets/14168765/62d6984e-e424-44e8-8f63-34b7421081a0)

Reply:
nice, looks great!!